server/models/house.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+const slug = require('slug');
+mongoose.Promise = global.Promise;
+
+const houseSchema = new Schema({
+  title: {
+    type: String,
+    trim: true,
+    required: 'Title is required'
+  },
+  slug: String,
```

Comment:
what is slug ?

Reply:
https://stackoverflow.com/a/19335586/4260745